content/tutorials/sbt/index.md

```diff
@@ -1,25 +1,27 @@
 ---
 title: "Getting started with sbt"
 menutitle: "sbt"
-description: "Learning how to setup FrontLine using sbt"
+description: "Learn how to setup Gatling Enterprise using sbt"
 date: 2021-03-19T17:03:31+01:00
 lastmod: 2021-03-19T17:03:31+01:00
 weight: 50
 ---
 
 {{< include introduction.md >}}
 
-There are multiple ways to create a Gatling artifact, which will be used by FrontLine to launch a run.
-You can use either a build tool (Maven, sbt, Gradle) or the Gatling bundle.
+## Download the sample Gatling simulation
 
-In this tutorial, we'll use the sbt build tool, so make sure you have sbt configured and Java installed.
-You'll use sbt from the terminal, but you can also do it easily with an IDE like IntelliJ.
+There are multiple ways to create a Gatling artifact, which will be used by Gatling Enterprise to launch a run.
+You can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.
```

Comment:
```suggestion
You can use either a build tool (Maven, SBT, Gradle) or the Gatling bundle.
```
Notes:
- On the official SBT website, it's spelled "sbt" (all lowercase), but I think in most of the docs we use "SBT" (all uppercase). Let's at least be consistent.
- Same thing for the other tutorials.